src/Animations/babylon.runtimeAnimation.ts

```diff
@@ -272,27 +274,7 @@
                         this._originalBlendValue = originalValue;
                     }
                 }
-            }
-
-            if (weight !== -1.0) {

```

Comment:
This used to get the original value only when weight is not -1.0. Is the original value intended to be the original value before blending? My new code does not check weight because I want to be able to reset the value back to the original value before setValue is ever called. Should I use separate variables for this?

Reply:
Unfortunately it is a bit trickier as the current value read and store in the original value is the value of the target at current time.
If an animation was already animating the target the you will get the current value computed from the animation.

This is why I think we should just get animation.keys[0].value (which was the intend of animate(0) but I think we can safely use first value